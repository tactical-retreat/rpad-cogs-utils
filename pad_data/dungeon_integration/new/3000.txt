Enoch Descended! - The Scribe-Legend Plus

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 10
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
14175 Damage - Change 2nd column to Fire orbs, Deal 150% damage (1 time use)

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 10
Even my meow will destroy!
8520 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
5964 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
12780 Damage - Change 2nd row to Dark orbs, Deal 150% damage (1 time use)

stage 1

760 - Sacred Dragon Beast, Angelion @ level 5
Halve Light & Wood
Reduce damage from Wood, Light by 50%
Holy Tree Defense
Voids status ailments for 999 turns
----- Turn 1 -----
Tail Wall
Reduce damage from all sources by 50% for 1 turns (1 time use)
----- HP <= 100 -----
Heavenly Wing Palm
16750 Damage - Change Dark to Heal, Deal 150% damage (50% chance)
Heavenly Wing Kick
16750 Damage - Change 4th, 5th row to Wood, Light orbs, Deal 150% damage
----- HP <= 30 -----
Holy Tree Vine
22333 Damage - Bind 2 random for 1~3 turns, Deal 200% damage

stage 2

625 - Wood Dominion Michael @ level 5
Protective Field
Voids status ailments for 4 turns
----- Execute below actions in order repeatedly -----
Sealing Field
Bind 1 both leaders for 4 turns
Soul Scales
3243 Damage - Deal 50% damage
Soul Scales
6486 Damage - Deal 100% damage
Soul Scales
12972 Damage - Deal 200% damage
Soul Scales
51888 Damage - Deal 800% damage

stage 3

1928 - Awoken Venus @ level 5
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 5
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns (1 time use)
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
161250 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 5
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns (1 time use)
----- Turn 1 -----
Corsier
Player -50% HP (1 time use)
----- Turn 1 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 2 -----
Arentillia
14175 Damage - Deal 140% damage (2 hits, 70% each) (1 time use)
----- Turn 2 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 3 -----
Goudenbel + Cure
Blind all orbs on the board + Player recover 100% hp (1 time use)
----- Turn 4 -----
Another Fate + Arentillia
Change own attribute to random one of Dark, Light + Enemy action (1 time use)
----- Turn 4 -----
Apotheosis Omen + Arentillia
Change own attribute to Light + Enemy action (1 time use)
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns
----- Execute below actions in order repeatedly -----
Staff of Destiny
Unable to Fire orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Water orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Wood orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Light orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Dark orbs for 2 turns (20% chance)
Misheliente
25312 Damage - Delay active skills by 1 turns, Deal 250% damage

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 13
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
18742 Damage - Change 2nd column to Fire orbs, Deal 150% damage (1 time use)

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 13
Even my meow will destroy!
11265 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
7885 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
16897 Damage - Change 2nd row to Dark orbs, Deal 150% damage (1 time use)

stage 1

760 - Sacred Dragon Beast, Angelion @ level 7
Halve Light & Wood
Reduce damage from Wood, Light by 50%
Holy Tree Defense
Voids status ailments for 999 turns
----- Turn 1 -----
Tail Wall
Reduce damage from all sources by 50% for 1 turns (1 time use)
----- HP <= 100 -----
Heavenly Wing Palm
24522 Damage - Change Dark to Heal, Deal 150% damage (50% chance)
Heavenly Wing Kick
24522 Damage - Change 4th, 5th row to Wood, Light orbs, Deal 150% damage
----- HP <= 30 -----
Holy Tree Vine
32696 Damage - Bind 2 random for 1~3 turns, Deal 200% damage

stage 2

625 - Wood Dominion Michael @ level 7
Protective Field
Voids status ailments for 4 turns
----- Execute below actions in order repeatedly -----
Sealing Field
Bind 1 both leaders for 4 turns
Soul Scales
4747 Damage - Deal 50% damage
Soul Scales
9495 Damage - Deal 100% damage
Soul Scales
18991 Damage - Deal 200% damage
Soul Scales
75965 Damage - Deal 800% damage

stage 3

1928 - Awoken Venus @ level 7
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 7
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns (1 time use)
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
236070 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 7
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns (1 time use)
----- Turn 1 -----
Corsier
Player -50% HP (1 time use)
----- Turn 1 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 2 -----
Arentillia
20752 Damage - Deal 140% damage (2 hits, 70% each) (1 time use)
----- Turn 2 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 3 -----
Goudenbel + Cure
Blind all orbs on the board + Player recover 100% hp (1 time use)
----- Turn 4 -----
Another Fate + Arentillia
Change own attribute to random one of Dark, Light + Enemy action (1 time use)
----- Turn 4 -----
Apotheosis Omen + Arentillia
Change own attribute to Light + Enemy action (1 time use)
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns
----- Execute below actions in order repeatedly -----
Staff of Destiny
Unable to Fire orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Water orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Wood orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Light orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Dark orbs for 2 turns (20% chance)
Misheliente
37057 Damage - Delay active skills by 1 turns, Deal 250% damage

Enoch Descended! - The Scribe-Mythical

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 10
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
14175 Damage - Change 2nd column to Fire orbs, Deal 150% damage (1 time use)

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 10
Even my meow will destroy!
8520 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
5964 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
12780 Damage - Change 2nd row to Dark orbs, Deal 150% damage (1 time use)

stage 1

760 - Sacred Dragon Beast, Angelion @ level 5
Halve Light & Wood
Reduce damage from Wood, Light by 50%
Holy Tree Defense
Voids status ailments for 999 turns
----- Turn 1 -----
Tail Wall
Reduce damage from all sources by 50% for 1 turns (1 time use)
----- HP <= 100 -----
Heavenly Wing Palm
16750 Damage - Change Dark to Heal, Deal 150% damage (50% chance)
Heavenly Wing Kick
16750 Damage - Change 4th, 5th row to Wood, Light orbs, Deal 150% damage
----- HP <= 30 -----
Holy Tree Vine
22333 Damage - Bind 2 random for 1~3 turns, Deal 200% damage

stage 2

625 - Wood Dominion Michael @ level 5
Protective Field
Voids status ailments for 4 turns
----- Execute below actions in order repeatedly -----
Sealing Field
Bind 1 both leaders for 4 turns
Soul Scales
3243 Damage - Deal 50% damage
Soul Scales
6486 Damage - Deal 100% damage
Soul Scales
12972 Damage - Deal 200% damage
Soul Scales
51888 Damage - Deal 800% damage

stage 3

1928 - Awoken Venus @ level 5
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 5
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns (1 time use)
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
161250 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 5
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns (1 time use)
----- Turn 1 -----
Corsier
Player -50% HP (1 time use)
----- Turn 1 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 2 -----
Arentillia
14175 Damage - Deal 140% damage (2 hits, 70% each) (1 time use)
----- Turn 2 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 3 -----
Goudenbel + Cure
Blind all orbs on the board + Player recover 100% hp (1 time use)
----- Turn 4 -----
Another Fate + Arentillia
Change own attribute to random one of Dark, Light + Enemy action (1 time use)
----- Turn 4 -----
Apotheosis Omen + Arentillia
Change own attribute to Light + Enemy action (1 time use)
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns
----- Execute below actions in order repeatedly -----
Staff of Destiny
Unable to Fire orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Water orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Wood orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Light orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Dark orbs for 2 turns (20% chance)
Misheliente
25312 Damage - Delay active skills by 1 turns, Deal 250% damage

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 13
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
18742 Damage - Change 2nd column to Fire orbs, Deal 150% damage (1 time use)

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 13
Even my meow will destroy!
11265 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
7885 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
16897 Damage - Change 2nd row to Dark orbs, Deal 150% damage (1 time use)

stage 1

760 - Sacred Dragon Beast, Angelion @ level 7
Halve Light & Wood
Reduce damage from Wood, Light by 50%
Holy Tree Defense
Voids status ailments for 999 turns
----- Turn 1 -----
Tail Wall
Reduce damage from all sources by 50% for 1 turns (1 time use)
----- HP <= 100 -----
Heavenly Wing Palm
24522 Damage - Change Dark to Heal, Deal 150% damage (50% chance)
Heavenly Wing Kick
24522 Damage - Change 4th, 5th row to Wood, Light orbs, Deal 150% damage
----- HP <= 30 -----
Holy Tree Vine
32696 Damage - Bind 2 random for 1~3 turns, Deal 200% damage

stage 2

625 - Wood Dominion Michael @ level 7
Protective Field
Voids status ailments for 4 turns
----- Execute below actions in order repeatedly -----
Sealing Field
Bind 1 both leaders for 4 turns
Soul Scales
4747 Damage - Deal 50% damage
Soul Scales
9495 Damage - Deal 100% damage
Soul Scales
18991 Damage - Deal 200% damage
Soul Scales
75965 Damage - Deal 800% damage

stage 3

1928 - Awoken Venus @ level 7
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 7
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns (1 time use)
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
236070 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 7
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns (1 time use)
----- Turn 1 -----
Corsier
Player -50% HP (1 time use)
----- Turn 1 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 2 -----
Arentillia
20752 Damage - Deal 140% damage (2 hits, 70% each) (1 time use)
----- Turn 2 -----
Sanctuary
Reduce damage from all sources by 50% for 10 turns (1 time use)
----- Turn 3 -----
Goudenbel + Cure
Blind all orbs on the board + Player recover 100% hp (1 time use)
----- Turn 4 -----
Another Fate + Arentillia
Change own attribute to random one of Dark, Light + Enemy action (1 time use)
----- Turn 4 -----
Apotheosis Omen + Arentillia
Change own attribute to Light + Enemy action (1 time use)
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns
----- Execute below actions in order repeatedly -----
Staff of Destiny
Unable to Fire orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Water orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Wood orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Light orbs for 2 turns (20% chance)
Staff of Destiny
Unable to Dark orbs for 2 turns (20% chance)
Misheliente
37057 Damage - Delay active skills by 1 turns, Deal 250% damage

