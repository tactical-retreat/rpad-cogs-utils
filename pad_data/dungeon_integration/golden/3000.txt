Enoch Descended! - The Scribe-Legend Plus

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 10
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
14175 Damage - Change 2nd column to Fire orbs, Deal 150% damage

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 10
Even my meow will destroy!
8520 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
5964 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
12780 Damage - Change 2nd row to Dark orbs, Deal 150% damage

stage 1

760 - Sacred Dragon Beast, Angelion @ level 5
Halve Light & Wood
Reduce damage from Wood, Light by 50%

stage 2

625 - Wood Dominion Michael @ level 5
Protective Field
Voids status ailments for 4 turns

stage 3

1928 - Awoken Venus @ level 5
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 5
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 70 -----
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
161250 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 5
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 13
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
18742 Damage - Change 2nd column to Fire orbs, Deal 150% damage

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 13
Even my meow will destroy!
11265 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
7885 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
16897 Damage - Change 2nd row to Dark orbs, Deal 150% damage

stage 1

760 - Sacred Dragon Beast, Angelion @ level 7
Halve Light & Wood
Reduce damage from Wood, Light by 50%

stage 2

625 - Wood Dominion Michael @ level 7
Protective Field
Voids status ailments for 4 turns

stage 3

1928 - Awoken Venus @ level 7
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 7
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 70 -----
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
236070 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 7
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns

Enoch Descended! - The Scribe-Mythical

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 10
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
14175 Damage - Change 2nd column to Fire orbs, Deal 150% damage

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 10
Even my meow will destroy!
8520 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
5964 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
12780 Damage - Change 2nd row to Dark orbs, Deal 150% damage

stage 1

760 - Sacred Dragon Beast, Angelion @ level 5
Halve Light & Wood
Reduce damage from Wood, Light by 50%

stage 2

625 - Wood Dominion Michael @ level 5
Protective Field
Voids status ailments for 4 turns

stage 3

1928 - Awoken Venus @ level 5
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 5
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 70 -----
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
161250 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 5
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns

stage 0

2305 - Red-Eyed Ghost, Red Cap @ level 13
Mwaha!
Delay active skills by 1~4 turns
----- Turn 1 -----
Axe of Madness
Player -80% HP
----- Turn 2 -----
Fire Swing
18742 Damage - Change 2nd column to Fire orbs, Deal 150% damage

2313 - Demon-Clawed Monster Cat, Cath Palug @ level 13
Even my meow will destroy!
11265 Damage - Deal 100% damage
----- Turn 1 -----
Innate Catastrophe
7885 Damage - Blind all orbs on the board, Deal 70% damage
----- Turn 2 -----
Darkside Claw
16897 Damage - Change 2nd row to Dark orbs, Deal 150% damage

stage 1

760 - Sacred Dragon Beast, Angelion @ level 7
Halve Light & Wood
Reduce damage from Wood, Light by 50%

stage 2

625 - Wood Dominion Michael @ level 7
Protective Field
Voids status ailments for 4 turns

stage 3

1928 - Awoken Venus @ level 7
*****
Reduce damage from Water, Light by 50%
Be gentle with me
Movetime -2s for 5 turns
----- HP <= 100 -----
This is just the beginning
Movetime -3s for 3 turns

stage 4

1827 - Red-Winged Star Angel, Rozuel @ level 7
Water damage halved
Reduce damage from Water by 50%
Flare Bomb + Flashburn
Spawn Bomb orbs in the specified positions + Blind orbs in specific positions for 1 turns
----- HP <= 100 -----
Healing Hymn + Scarlet Azure Blade
Change Random to Heal + Change 2nd, 3rd row to Fire, Water orbs
----- HP <= 70 -----
Light Punishment
Jammer skyfall +20% for 5 turns
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 70 -----
Angel's Holy Grail + Battle Anthem
Enemy recover 100% hp + Increase damage to 150% for the next 999 turns
----- HP <= 15 -----
Lotus Sky Blades
236070 Damage - Change 1st, 3rd, 5th row to Fire orbs, 2nd, 4th row to Water orbs, Deal 1500% damage

stage 5

3600 - Heaven Scribe, Enoch @ level 7
Light Dark Halved
Reduce damage from Light, Dark by 50%
Resolve
Survive attacks with 1 HP when HP > 76%
Saint's Robe + Staff of Destiny
Voids status ailments for 999 turns + Unable to Heal orbs for 3 turns
----- HP <= 35 -----
Sage Wisdom
Absorb combo <= 6 damage for 999 turns
----- HP <= 1 -----
Divine Cure + Sage Wisdom
Enemy recover 15% hp + Absorb combo <= 6 damage for 999 turns

